Tasks/PublishBuildArtifacts/Invoke-Robocopy.ps1

```diff
@@ -6,8 +6,11 @@ param(
     [Parameter(Mandatory = $true)]
     [string]$Target,
 
-	[Parameter(Mandatory = $true)]
-    [int]$ParallelCount)
+    [Parameter(Mandatory = $true)]
+    [int]$ParallelCount,
+
+    [Parameter(Mandatory = $false)]
```

Comment:
Why does this need to be an input?

Reply:
This script is invoked by the typescript file, it's not run by the powershell handler directly.  We need to inform the PS1 script to know which file to copy.

Reply:
Ah, I see it now, strange!